include/private/tdympfaoimpl.h

```diff
@@ -4,7 +4,6 @@
 #include <petsc.h>
 
 PETSC_EXTERN PetscErrorCode TDyMPFAOInitialize(TDy);
```

Comment:
Just to check: do you want external linkage on this function? Does one of the drivers use it?

Reply:
Good catch. I have fixed this now.